pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -158,7 +167,14 @@ func (s *Installer) Install(ctx context.Context, bundle releasev1alpha1.Tinkerbe
 		},
 		rufio: map[string]interface{}{
 			deploy: true,
-			image:  bundle.Rufio.Image.URI,
+			image:  bundle.TinkerbellStack.Rufio.Image.URI,
+		},
+		loadBalancer: map[string]interface{}{
+			"enabled": s.loadBalancer,
+			"ip":      tinkerbellIP,
```

Comment:
nit: It would be nice to call this tinkerbellVIP or tinkerbellServiceVIP, just so it is clear that it is a VIP for target cluster tinkerbell stack.

Reply:
This function is generic to both installing tinkerbell on bootstrap and on workload.
Calling this `tinkerbellVIP` would make it specific to installing this on workload since on bootstrap, it won't be a vip